src/main/java/core/basesyntax/ArrayList.java

```diff
@@ -1,48 +1,115 @@
 package core.basesyntax;
 
+import java.util.NoSuchElementException;
+
 public class ArrayList<T> implements List<T> {
+    private static final int DEFAULT_CAPACITY = 10;
+    private static final double CAPACITY_MULTIPLY_BY = 1.5;
+    private static final String NEGATIVE_INDEX_ERROR = "Index should be natural number";
+    private static final String INDEX_OUT_OF_SIZE_ERROR = "Index out of list length";
+
+    private T[] data;
```

Comment:
make it final

Reply:
But then I won't be able to overwrite it with the new capacity